storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -2053,7 +2045,6 @@ bool write_backup_config_file() {
     << innodb_checksum_algorithm_names[srv_checksum_algorithm] << "\n"
     << "innodb_log_checksums=" << srv_log_checksums << "\n"
     << "innodb_data_file_path=" << innobase_data_file_path << "\n"
-    << "innodb_log_files_in_group=" << srv_n_log_files << "\n"
     << "innodb_log_file_size=" << innobase_log_file_size << "\n"
```

Comment:
we might have to consider when both the old (log)parameter and the new redo_log_capacity parameters are set.

Reply:
@satya-bodapati same here, this is no longer necessary.